### ISSUE TITLE
Search by ID

### ISSUE THREAD

**Issue by SonomaGTS** [bonukai/MediaTracker#436]
It would be nice if you could include the "show_id" within the search parameters since I am assuming you are still using the TMDB API to fetch? TV shows in particular are horrible to look up sometimes. Since the url of every TMDB listed show and movie contains a unique ID which is the "show_id" via the api it would make things a little easier.

**Comment by bonukai** [comment]
I'm not sure I understand. Are you asking to be able to search, for example for Lost, by typing [tt0411008](https://www.imdb.com/title/tt0411008/) (IMDb id) or [4607](https://www.themoviedb.org/tv/4607-lost) (TMDB id)?

**Comment by SonomaGTS** [comment]
> I'm not sure I understand. Are you asking to be able to search, for example for Lost, by typing [tt0411008](https://www.imdb.com/title/tt0411008/) (IMDb id) or [4607](https://www.themoviedb.org/tv/4607-lost) (TMDB id)?

Yes. Due to many shows and movies now having the same name or being too similar to other names sometimes when I go to add a tv show or movie if the result doesn't 500 error it still yields a sea of wrong results. The TV show "See" for an example was initially difficult for me to add. It would be nice as an alternative means of searching for content whereas there would be no way to yield multiple or wrong results.
